style.css

```diff
@@ -13,10 +13,38 @@ body {
 
 .background {
   padding-bottom: 60%;
-  background-image: url(images/geometry.png);
   background-size: cover;
   background-position: bottom;
   background-repeat: no-repeat;
+  margin: -1px 0 -2px 0;
+}
+
+.img1 {
+  background-image: url(../images/geometry.png);
+}
+
+.img2 {
+  background-image: url(../images/1.png);
+}
+
+.img3 {
+  background-image: url(../images/2.png);
+}
+
+.img4 {
+  background-image: url(images/3.png);
+}
+
+.img5 {
+  background-image: url(../images/4.png);
+}
+
+.img6 {
+  background-image: url(../images/5.png);
+}
+
+.img7 {
+  background-image: url(../images/6.png);
```

Comment:
- [ ] Please add `./` instead of `../` before all URL paths because it is not working online maybe it's working on your local machine but online it's not.
for example `background-image: url(./images/6.png);`